bindings/runner/src/common.rs

```diff
@@ -686,38 +686,19 @@ pub fn run_holochain_conductor<SV: UserValuesConstraint>(
 
     let config = ctx.get_mut().take_holochain_config().build()?;
 
-    ctx.get_mut().holochain_runner =
-        ctx.runner_context()
-            .executor()
-            .execute_in_place(async move {
-                HolochainRunner::run(&config)
-                    .await
-                    .map(Some)
-                    .map_err(|mut err| {
-                        log::trace!("Error whilst starting conductor so cleaning up directory");
```

Comment:
this code was a mess and couldn't even get formatted by rust fmt. Moved to a separate function.